src/themes/_common/player/player.scss

```diff
@@ -121,3 +121,7 @@
     overflow: initial;
 }
 
+.#{$cssplayer}-hide-outstream {
+    visibility: hidden;
+    right: -500vw;
```

Comment:
no need to shift position.  We don't know which position the player can be.  It could be positioned in bottom left instead so something like this could cause problems. visibility hidden should be good enough

Reply:
500vw, means -500% of width of the screen so it could feet on all screen widths in all player position. With only visibility, it could cover some elements in the right site of the page like some banner.

Reply:
so if i position this player to the left side and this plays, it wont transition from the right side?

Reply:
If we're setting left side, we're setting `left: value`, and top or bottom. We can't set right correctly on that case. So applying `transition: all ...` will not affect on `right` property.